.github/workflows/ci.yaml

```diff
@@ -64,8 +64,8 @@ jobs:
       # Don't remove it! We shouldn't allow CI to pass if there are any changes not committed after running gen/lint/test
       - name: Fail on dirty
         run: |
-          if ! git status --short || ! git diff --quiet; then
-            git --no-pager diff -- . :^vendor
+          if ! git status --short ':(exclude)vendor' ':(exclude)src/generated' || ! git diff --quiet ':(exclude)vendor' ':(exclude)src/generated'; then
```

Comment:
For some reason I have to explicitly add those exclusions

Reply:
If you'll add rust codegen into the `just gen` it'll run `just lint` after that automatically which injects license header and so you'll have generated files in a right shape for CI to not complain

Reply:
done